common/main/java/com/couchbase/lite/internal/core/C4Collection.java

```diff
@@ -293,7 +293,17 @@ public void createVectorIndex(
 
     @NonNull
     public FLValue getIndexesInfo() throws LiteCoreException {
-        return FLValue.getFLValue(withPeerOrThrow(impl::nGetIndexesInfo));
+        return withPeerOrThrow(peer -> FLValue.getFLValue(impl.nGetIndexesInfo(peer)));
+    }
+
+    @GuardedBy("Database.getDbLock()")
+    @Nullable
+    public C4Index getIndex(@NonNull String name) throws LiteCoreException {
+        // ??? need to check collection for validity?
```

Comment:
Should be the same as what the other methods in the Collection class are doing.

Reply:
Yes.  fixed.  Thx!